pkg/fakerp/config.go

```diff
@@ -84,6 +86,19 @@ func GetPluginTemplate() (*pluginapi.Config, error) {
 	return template, nil
 }
 
+// GetContext returns enriched context for plugin initiation
+func GetContext(duration time.Duration) (context.Context, context.CancelFunc) {
```

Comment:
why do we hard code a timeout in any of this PR? can't we have the caller decide what they want? can this function be `func(ctx context.Context) context.Context` ?